### PR TITLE
Add debugtiler.sh

### PR DESCRIPTION
**Pull request by kdeloach**
May be usuful for troubleshooting tiler issues. Copied over from NYC
TreesCount!

### CODE REVIEW

**Comment by mmcfarland** [comment]
+1 Tested.  Great output, too.

File: src/tiler/package.json [hunk 1]
```diff
@@ -6,8 +6,12 @@
   "dependencies": {
     "windshaft": "0.36.0"
   },
+  "devDependencies": {
+    "supervisor": "^0.6.0"
```
Comment (hectcastro) [inline src/tiler/package.json]: According to [this project's `README`](https://github.com/isaacs/node-supervisor/blob/master/README.md), it is no longer being maintained. Do we have any better alternatives?

Comment (kdeloach) [inline src/tiler/package.json]: Open to suggestions, but this isn't mission critical (more of a convenience).

Comment (hectcastro) [inline src/tiler/package.json]: A thing I've wanted to evaluate to try and solve this problem in a more general way, but haven't had time yet is [Watchman](https://facebook.github.io/watchman/).

Comment (kdeloach) [inline src/tiler/package.json]: Added an issue https://github.com/WikiWatershed/model-my-watershed/issues/270

**Pull request merged**